R/confirmatoryfactoranalysis.R

```diff
@@ -898,6 +941,38 @@ confirmatoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
     vi$setData(vidat)
     vi$dependOn(optionsFromObject = jrobject)
   }
+
+
+  # Thresholds
+  if ("|" %in% pei$op) {
+
+    # Manifest variable intercepts
+    jrobject[["Thresholds"]] <- th <- createJaspTable(title = gettext("Thresholds"))
+    if (!is.null(footnote)) th$addFootnote(footnote)
+
+    th$addColumnInfo(name = "lhs",    title  = gettext("Indicator threshold"),  type = "string", combine = TRUE)
+    th$addColumnInfo(name = "est",    title  = gettext("Estimate"),   type = "number", format = "sf:4;dp:3")
+    th$addColumnInfo(name = "se",     title  = gettext("Std. Error"), type = "number", format = "sf:4;dp:3")
+    th$addColumnInfo(name = "z",      title  = gettext("z-value"),    type = "number", format = "sf:4;dp:3")
+    th$addColumnInfo(name = "pvalue", title  = gettext("p"),          type = "number", format = "dp:3;p:.001")
+
+    th$addColumnInfo(name = "ci.lower", title = gettext("Lower"), type = "number", format = "sf:4;dp:3",
+                     overtitle = gettextf("%s%% Confidence Interval", options$ciLevel * 100))
+    th$addColumnInfo(name = "ci.upper", title = gettext("Upper"), type = "number", format = "sf:4;dp:3",
+                     overtitle = gettextf("%s%% Confidence Interval", options$ciLevel * 100))
+
+    if (options$standardized != "none")
+      th$addColumnInfo(name = paste0("std.", standardization), title = gettextf("Std. Est. (%s)", standardization),
+                       type = "number", format = "sf:4;dp:3")
+
+    # add data
+    thdat <- pei[pei$op == "|", colSel[-c(2, 3)]]
+    thdat$lhs <- paste(thdat$lhs, "|", pei$rhs[pei$op == "|"])
```

Comment:
I think t's nicer if you show these in two separate columns ("Indicator", "Threshold") with `combine = TRUE`...